### DISCUSSION TITLE
perchè non si può aggiustare?

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#52] [Generale]
Non ho potuto fare a meno di notare che quando inizio il tutorial, un secondo prima di palare con jack, mi appare un riquadro.
doce c'è scritto :+1: 
"usa il mouse per guardarti attorno"
"usa i tasti W,S,A,D"

sotto poi c'è scritto "cliccare per" e dopo un lungo spazio c'è scritto "con"
presumo che il "con" stia per "continuare".

potete dirmi quale è il file in questione dove trovo quella scritta? per favore

**Comment by crazyLillo** [comment]
ho trovato quello che cercavo e ho provato a modificarlo e va bene, ma voglio capire meglio e approfondire del perchè la scritta non ci sta,
e forse ho capito anche perchè.

**Reply by crazyLillo** [reply]
**Io l'ho trovato così**

![1](https://user-images.githubusercontent.com/96891196/149397578-3fc2c3b4-0d1d-4210-864c-94f23c12c76b.jpg)

**Ma adesso è così, uguale a quello inglese**

![2](https://user-images.githubusercontent.com/96891196/149397727-8bb533d6-95f7-471a-a817-5297b5447754.jpg)


**ma potrebbe essere anche così se necessario (ma la scritta continua sotto al piccolo riquadro).**

![3](https://user-images.githubusercontent.com/96891196/149397967-2a3e6f47-0dfd-4868-ac5f-28d3c0159739.jpg)


**Ditemi voi come andrebbe meglio, posso modificarla.**

**Reply by NarakuITA** [reply]
La seconda.

**Reply by crazyLillo** [reply]
la terza l'ho fatta solo in caso volevate inserire una frase piu lunga

si, è molto meglio la seconda.

..e questi nello zip sono più di un centinaio di file "aggiustati", che vanno in vdata\signs.

non sono tutti i file della cartella signs , ma solo tutti i popup del tutorial, che sono più di 100.

ho notato altre cose, ma tempo al tempo.

[signs.zip](https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/files/7866452/signs.zip)

**Reply by tesivo** [reply]
La seconda.
Ottimo che hai corretto altri di questi messaggi. Mi sa che mi erano sfuggiti ;)

**Comment by crazyLillo** [comment]
si ok per la seconda, anche qui si potrebbe allargare la finestra (ma sono centinaia da modificare...)
